astrolabe/validator.py

```diff
@@ -98,7 +98,7 @@ def run_test_expecting_error(self, driver_workload):
         try:
             try:
                 stats = subprocess.stop()
-            except PrematureExitError:
+            except (PrematureExitError, PermissionError):
```

Comment:
The expected `PrematureExitError` here was showing up as a `PermissionError` for me when the validator attempted to kill an already-terminated process.

Reply:
Could you add this as a comment in the code please so that whoever looks at this section in the future is aware of this case?